### PR TITLE
sendmail 8.17.1 - added new feature MTA-STS, elliptic curves (TLS), …

### PR DESCRIPTION
**Pull request by fritzkink**
…DANE and SMTPUTF8 support
- Experimental support for SMTPUTF8 (EAI, see RFC 6530-6533)
- Experimental support for SMTP MTA Strict Transport Security
- support for DANE (see RFC 7672 et.al.)
- TLS_EC to enable the use of elliptic curve cryptography in STARTTLS

### CODE REVIEW

File: components/mail/sendmail/Makefile [hunk 1]
```diff
@@ -182,22 +190,24 @@ COMPONENT_TEST_TRANSFORMS += \
...
 
 # common targets
-build:		$(BUILD_32_and_64)
+BUILD_TARGET:		$(BUILD_32_and_64)
 
-install:	$(INSTALL_32_and_64)
+INSTALL_TARGET:	$(INSTALL_32_and_64)
 
-test:		$(TEST_32_and_64)
+TEST_TARGET:		$(TEST_32_and_64)
```
Comment (AndWac) [inline components/mail/sendmail/Makefile]: You can replace these with a simple BUILD_BITS= 32_and_64 (or, if you prefer 64 bit apps 64_and_32).

Comment (fritzkink) [reply components/mail/sendmail/Makefile]: great idea. I've adapeted

File: components/mail/sendmail/Makefile [hunk 2]
```diff
@@ -21,27 +21,27 @@
...
+COMPONENT_NAME=         sendmail
+COMPONENT_VERSION=      8.17.1
+COMPONENT_REVISION=     1
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_PROJECT_URL=  ftp://ftp.sendmail.org/pub/sendmail/
+COMPONENT_ARCHIVE=      $(COMPONENT_NAME).$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:04bc76b6c886e6d111be7fd8daa32b8ce00128a288b6b52e067bc29f3854a6e6
+COMPONENT_ARCHIVE_URL=  $(COMPONENT_PROJECT_URL)$(COMPONENT_ARCHIVE)
 
 TPNO=			23958
```
Comment (AndWac) [inline components/mail/sendmail/Makefile]: Please remove the TPNO (and also from manifest file if referenced), it's an Oracle thing.

**Comment by pyhalov** [comment]
"snedmail" in commit message

**Pull request merged**